web/src/components/tasks/business-formation/DbaFormationPaginator.tsx

```diff
@@ -190,7 +198,11 @@ export const DbaFormationPaginator = (): ReactElement => {
           text={state.dbaContent.Authorize.callToActionText}
           onClick={(): void => setShowCtaModal(true)}
         >
-          <HelpButton />
+          <HelpButton
+            onClick={(): void => {
+              analytics.event.business_formation_help_button.click.open_live_chat();
+            }}
+          />
```

Comment:
So the DBA formation paginator should also have tests associated probably for analytics I think.

The issue here being I started doing this then realized that there are a bunch of missing tests and for this file and that because there is no file at the moment there a bunch of missing helper functions and its going to be a bit of work to get it done

So I made a follow up refactor friday ticket so I can deal with it in agregate. https://www.pivotaltracker.com/story/show/186377291

Reply:
The ticket is a great idea,

If the analytics is defined inside the component, you would only need to test firing inside the component.